Lib/ftplib.py

```diff
@@ -102,15 +103,15 @@ class FTP:
     file = None
     welcome = None
     passiveserver = 1
-    encoding = "latin-1"
 
-    def __init__(self, host='', user='', passwd='', acct='',
+    def __init__(self, host='', user='', passwd='', acct='', encoding='utf-8',
```

Comment:
This change is backward incompatible if someone uses positional arguments. Please add encoding as a keyword-only parameter. Same remark for FTP_TLS.